app/controllers/hiring_staff/base_controller.rb

```diff
@@ -34,21 +34,26 @@ def current_user
   end
 
   def check_user_last_activity_at
-    return redirect_to dsi_logout_url if current_user&.last_activity_at.blank?
+    return redirect_to logout_endpoint if current_user&.last_activity_at.blank?
```

Comment:
This could be another before_filter

Reply:
`check_user_last_activity_at` is a before_action. Do you want me to split it in two?

Reply:
I agree with you